.pre-commit-config.yaml

```diff
@@ -28,26 +28,19 @@ repos:
       - id: check-yaml
       - id: check-ast
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    # Ruff version (Used for linting)
-    rev: v0.7.4
+    rev: v0.8.6
     hooks:
       - id: ruff
-        args: [ --fix, --exit-non-zero-on-fix, --preview ]
```

Comment:
I think we have the `--exit-non-zero-on-fix` in there for the CI

Reply:
I agree with removing `--preview`: https://docs.astral.sh/ruff/preview/

Reply:
added back `--exit-non-zero-on-fix`